packages/server/lib/cloud/protocol.ts

```diff
@@ -394,6 +428,27 @@ export class ProtocolManager implements ProtocolManagerShape {
     this._errors = []
   }
 
+  reset (): void {
```

Comment:
Is this actually used in a `reset` scenario? Or should it be renamed to `release`, or `close`?

Reply:
Renamed to close: [`e619cb2` (#31239)](https://github.com/cypress-io/cypress/pull/31239/commits/e619cb23c7d692f22ef0e29370b9b5d8555e6a72)